setup.py

```diff
@@ -35,13 +35,19 @@ def load_requirements(use_case):
     },
     license="Apache2.0",
     classifiers=[
+        "Environment :: Console",
+        "Environment :: Web Environment",
         "Programming Language :: Python",
         "Programming Language :: Python :: 3",
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
-        "Development Status :: 3 - Alpha",
+        "Programming Language :: Python :: 3.10",
+        "Framework :: Flask",
+        "Development Status :: 5 - Production/Stable",
         "License :: OSI Approved :: Apache Software License",
         "Operating System :: OS Independent",
```

Comment:
This one can probably go then. ([possibly related issue](https://github.com/getsentry/sentry/issues/1903))

Reply:
Good catch, that was the idea.